test/blueprints/Do.Test.Blueprints.Service.Test/TestServiceNfrExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Newtonsoft.Json;
+
+namespace Do.Test;
+
+public static class TestServiceNfrExtensions
+{
+    public async static Task<dynamic?> DeserializeContentToDynamic(this HttpContent content)
```

Comment:
don't return `dynamic`, return `object` instead, whoever uses this can have the option to set it to a `dynamic` variable anyway.